### PR TITLE
Navbar Update

### PR DESCRIPTION
**Pull request by IC1101Virgo**
After a lot of try of this is what I have come up with.
Please tell me of any suggestions.
Thank you!

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -15,6 +15,35 @@
...
+                    About Us
+                  </a>
+                  <a class="item-nav navbar-item" href="#">
+                    Contact Us
+                  </a>
+                </div>
+              </div>
+          </nav>
+        </div>
+        <!-- Navigation Bar End -->
```
Comment (irshadjsr21) [inline public/index.html:46]: These changes should be inside the `common/navbar/navbar.js` file.

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]
Hi @IC1101Virgo , 

There are some changes that you need to make in order for it to be a react component. I recommend you to check out the `Resources` section of the `README.md` file to get familiar with React.

Please make the following changes in this PR.

1. The Girlscript logo is not being rendered.
2. You should not change the `public/index.html` file. The navbar HTML should be written in `common/navbar/navbar.js` file.
3. In the `Home.js` file, please import the `Navbar` component and render it.

File: public/index.html (hunk 1 above)
Comment (IC1101Virgo) [reply public/index.html:46]: ok sir. Will update and work on these.

**Comment by IC1101Virgo** [comment]
@irshadjsr21 Hello sir, I have been trying to fix this but I am unable to convert HTML code to JS also there is some issue with the logo image, actually the logo image was not provided in the image folder and I have tried to include a screenshot of the logo which does not provide with quality. 
I believe the navbar works fine with HTML code {Please correct me 😄 }
If you could merge my PR and open up follow issues to further enhance the Navbar would be great!
I am open to your suggestions  😸 
Thank you

**Comment by smilegupta** [comment]
@IC1101Virgo can you change the source branch to develop and resolve the merge conflicts and may send a fresh PR please

**Comment by IC1101Virgo** [comment]
@smilegupta ok I'll make a fresh PR 😄

**Pull request closed without merging**